### ISSUE TITLE
package not found

### ISSUE THREAD

**Issue by pangxiao2013** [shjung13/Standardized-max-logits#3]
Hello，can you tell me where to download ood_metrics packages?

**Comment by shjung13** [comment]
Hi @pangxiao2013, you can install the ood_metrics package from pip.
```pip install ood-metrics```
